Document zip MIME type on Windows (#34000)
mdn/content: files/en-us/web/http/basics_of_http/mime_types/common_types/index.md

| `.xml`          | XML                                                                                        | `application/xml` is recommended as of [RFC 7303](https://datatracker.ietf.org/doc/html/rfc7303#section-4.1) (section 4.1), but `text/xml` is still used sometimes. You can assign a specific MIME type to a file with `.xml` extension depending on how its contents are meant to be interpreted. For instance, an Atom feed is `application/atom+xml`, but `application/xml` serves as a valid default. |
| `.xul`          | XUL                                                                                        | `application/vnd.mozilla.xul+xml`                                                                                                                                                                                                                                                                                                                                                                         |

> | `.zip`          | ZIP archive                                                                                | `application/zip` is the standard, but beware that Windows uploads .zip with MIME type `application/x-zip-compressed`.                                                                                                                                                                                                                                                                                    |

Reviewer: ```suggestion
| `.zip`          | ZIP archive                                                                                | `application/zip` The standard. Note: Windows uploads .zip with MIME type `application/x-zip-compressed`.                                                                                                                                                                                                                                                                                    |
```
suggest making it shorter for the sake of the table.
